### DISCUSSION TITLE
Anon read connection

### DISCUSSION THREAD

**Discussion by 2547techno** [Chatterino/chatterino2#5455] [Ideas] [answered]
I'm not sure what the general sentiment is about this, but due to the recent change about max concurrent joins it would be nice to be able to join channels with an anon connection while still being able to send messages via helix.

Maybe as a different channel type, enabled via a setting similar to IRC channels? 
![image](https://github.com/Chatterino/chatterino2/assets/109011672/e8abaa49-6d0b-4a53-9dec-8ae6a48eacab)

Or maybe just as a global option? Not sure.

**Comment by Felanbird** [comment, marked as answer]
The general sentiment is no on the grounds of making it actually too easy to bypass channel bans, it is possible to do so with Anon mode already but we're not interested in allowing you to be logged in normally and also be given the easy accessibility to access your banned channels

**Reply by Felanbird** [reply]
And to follow-up on that, it's an intended Twitch restriction to lock you out of banned channels, the same way this new restriction limits your connections.

**Reply by 2547techno** [reply]
that's fair, just thought i'd ask anyway, thanks :)
